frontend/src/views/MyProgress/summary/WasteMeasureSummary.vue

```diff
@@ -1,11 +1,168 @@
 <template>
   <div class="fr-text">
-    Waste summary
+    <ul>
+      <li v-if="displayDiagnostic.hasWasteDiagnostic">
+        <v-icon color="primary" class="mr-2">$check-line</v-icon>
+        <div>
+          J’ai réalisé un diagnostic sur les causes probables de gaspillage alimentaire
+        </div>
+      </li>
+      <li v-else>
+        <v-icon color="primary" class="mr-2">$close-line</v-icon>
+        <div>
+          Je n’ai pas encore réalisé un diagnostic sur les causes probables de gaspillage alimentaire
+        </div>
+      </li>
+
+      <li v-if="displayDiagnostic.hasWastePlan">
+        <v-icon color="primary" class="mr-2">$check-line</v-icon>
+        <div>
+          J’ai mis en place un plan d’action adapté au diagnostic réalisé
+        </div>
+      </li>
+      <li v-else>
+        <v-icon color="primary" class="mr-2">$close-line</v-icon>
+        <div>
+          Je n’ai pas encore mis en place un plan d’action adapté au diagnostic réalisé
+        </div>
+      </li>
+
+      <li v-if="appliedWasteActions">
+        <v-icon color="primary" class="mr-2">$check-line</v-icon>
+        <div>
+          J’ai mis en place les actions suivantes :
+          <ul class="mt-2">
+            <li class="fr-text-xs mb-1" v-for="(action, index) in appliedWasteActions" :key="`${action}-${index}`">
+              {{ action }}
+            </li>
+          </ul>
+        </div>
+      </li>
+      <li v-else>
+        <v-icon color="primary" class="mr-2">$close-line</v-icon>
+        <div>
+          Je n’ai pas encore mis en place des actions concrètes contre le gaspillage
+        </div>
+      </li>
+
+      <li v-if="displayDiagnostic.hasWasteMeasures">
+        <v-icon color="primary" class="mr-2">$check-line</v-icon>
+        <div>
+          J’ai réalisé des mesures de mon gaspillage alimentaire :
+          <ul class="mt-2">
+            <li class="fr-text-xs mb-1" v-for="measure in wasteMeasures" :key="measure.label">
+              {{ measure.label }} :
+              <span class="font-weight-bold">{{ measure.value }}</span>
```

Comment:
```suggestion
              <span class="font-weight-bold ml-1">{{ measure.value }}</span>
```

![Screenshot 2023-10-25 at 16-06-18 ma cantine](https://github.com/betagouv/ma-cantine/assets/9282816/e37339ff-7826-409f-b5a6-c461742626fd)